Update protect-personal-data-azure.md

v-debshi opened the pull request:
Delete reference to medical information and reverse order of "customer and employee" at the request of Eric Tierling.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

v-debshi committed:
Update protect-personal-data-azure.md

Delete reference to medical information and reverse order of "customer and employee" at the request of Eric Tierling.
